x-pack/plugins/observability/public/pages/alerts/translations.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const ALERTS_BREADCRUMB_TEXT = i18n.translate(
+  'xpack.observability.breadcrumbs.alertsLinkText',
```

Comment:
We have this key in a lot of places under observability

Reply:
Do you mean to change the key or use the existing one?
I am going to remove this file and move translation in the index.tsx file for now.

Reply:
I meant to put this key in one place so all the pages using it can import it. But this could be in another PR